dao/src/main/java/org/thingsboard/server/dao/service/validator/QueueValidator.java

```diff
@@ -38,6 +40,8 @@ public class QueueValidator extends DataValidator<Queue> {
     @Autowired
     private TbTenantProfileCache tenantProfileCache;
 
+    private final Pattern queueTopicPattern = Pattern.compile("^[a-zA-Z0-9_.\\-]+$");
```

Comment:
You should move the pattern and the validation to the DataValidator class in order to reuse it in both QueueValidator and TenantProfileDataValidator. See DataValidator.doValidateEmail or similar.